docs/src/main/asciidoc/amazon-lambda.adoc

```diff
@@ -606,6 +606,74 @@ jar tvf target/function.zip
     libsunec.so
 ----
 
+== Deploy to AWS Lambda using a Container Image
+
+AWS Lambda supports creating your lambdas by referencing https://docs.aws.amazon.com/lambda/latest/dg/lambda-images.html[container images] rather than uploading ZIP files. This can have some benefits such as bypassing the size limit of the uploaded ZIP files. You can define lambda functions for both native builds as well as regular JVM builds.
+
+=== JVM container image
+
+For a regular JVM distribution you need to base your image off the official AWS Java base images. Below is an example of a Dockerfile that would create a container image from your Quarkus Lambda project. It assumes that `mvn package` has been executed and binaries are available in the `target/` directory:
+
+[source,Dockerfile]
+----
+FROM  public.ecr.aws/lambda/java:11
+
+ADD target/my-service-0.0.1-SNAPSHOT-runner.jar /var/task/lib/my-service.jar
+ADD target/lib/  /var/task/lib/
+
+CMD ["io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler::handleRequest"]
+----
+
+=== Native executable container image
+
+To create a lambda container image that uses the native executable we'll need to do things a little differently. In this case, we won't need to use the `java:11` base image from AWS but instead we'll use a special image that assumes that the runtime environment for the lambda is provided. The example below creates such a container. It assumes that a Maven build has been executed (such as `mvn package -Dnative=true`) and has generated the native binary into the `target/` directory. The binary needs to be named `bootstrap` and be placed in `/var/runtime/`:
+
+[source,Dockerfile]
+----
+FROM  public.ecr.aws/lambda/provided
+
+ADD target/my-service-0.0.1-SNAPSHOT-runner /var/runtime/bootstrap
+RUN chmod ugo+x /var/runtime/bootstrap
+
+CMD ["io.quarkus.amazon.lambda.runtime.QuarkusStreamHandler::handleRequest"]
+----
+
+=== Deploying a container image lambda
+
+Although it is out of scope for this guide, you can see how the above container images can be built and deployed to AWS in the instructions below, using the `docker` and `aws` command line tools.
+
```

Comment:
You can remove the part where you say that it's out of the scope of the guide. Guides aim to be as complete as possible so when you follow it, you end up with a working application. 
So in case of function/lambda, it is in the scope of the guide to deploy it to the function provider.

Reply:
Done